zhaquirks/tuya/ts0601_cover.py

```diff
@@ -373,6 +373,7 @@ class TuyaMoesCover0601(TuyaWindowCover):
             ("_TZE200_7eue9vhc", "TS0601"),
             ("_TZE200_bv1jcqqu", "TS0601"),
             ("_TZE200_nw1r9hp6", "TS0601"),
+            ("_TZE200_9p5xmj5r", "TS0601")
```

Comment:
For the `pre-commit` check:
```suggestion
            ("_TZE200_9p5xmj5r", "TS0601"),
```

(Python 3.12 test failure is expected for now. Should be fixed on Tuesday or so.)

Reply:
@TheJulianJES actually, it looks like this wont work. it does control it but the position is inverted. i tried adding this model to all of the current inverted position classes and none of them seem to work

Reply:
Is ZHA loading the quirk for your device OK or is it mismatching the signature ?

Reply:
honestly i am very very new to the concept of zigbee quirks and how zigbee works so I don't know how to answer that question. what i do know is that when i make a local copy of this file then add the model mine reports, it does control up/down but it just reports incorrectly that the blind is up when its down and down when its up. If you give me a clue how to confirm what you need i am competent enough to do it.

Reply:
If the system is mashing it OK and loading the quirk yu is getting somthing like that on the device cards device info: `Quirk: zhaquirks.ikea.symfonisk2.IkeaSymfoniskGen2v2` (bu with your quirk name and not IKEA) and also in the diagnostic. 
If not loaded is normally somthing that is not OK in the device signature in the quirk that is not the same as your device.

Reply:
Actually, if I have it working correctly now with the below custom quirk python file loaded:
`"""Tuya based cover and blinds."""
from zigpy.profiles import zha
from zigpy.zcl.clusters.general import Basic, Groups, Identify, OnOff, Ota, Scenes, Time

from zhaquirks.const import (
    DEVICE_TYPE,
    ENDPOINTS,
    INPUT_CLUSTERS,
    MODELS_INFO,
    OUTPUT_CLUSTERS,
    PROFILE_ID,
)
from zhaquirks.tuya import (
    TuyaManufacturerWindowCover,
    TuyaManufCluster,
    TuyaWindowCover,
    TuyaWindowCoverControl,
)

class TuyaMoesCover0601(TuyaWindowCover):
    """Tuya blind controller device."""
    
    tuya_cover_inverted_by_default = True

    signature = {
        # "node_descriptor": "NodeDescriptor(byte1=2, byte2=64, mac_capability_flags=128, manufacturer_code=4098,
        #                    maximum_buffer_size=82, maximum_incoming_transfer_size=82, server_mask=11264,
        #                    maximum_outgoing_transfer_size=82, descriptor_capability_field=0)",
        # "endpoints": {
        # "1": { "profile_id": 260, "device_type": "0x0051", "in_clusters": [ "0x0000", "0x0004","0x0005","0xef00"], "out_clusters": ["0x000a","0x0019"] }
        # },
        # "manufacturer": "_TZE200_zah67ekd",
        # "model": "TS0601",
        # "class": "zigpy.device.Device"
        # }
        MODELS_INFO: [
            ("_TZE200_zah67ekd", "TS0601"),
            ("_TZE200_nueqqe6k", "TS0601"),
            ("_TZE200_gubdgai2", "TS0601"),
            ("_TZE200_5sbebbzs", "TS0601"),
            ("_TZE200_hsgrhjpf", "TS0601"),
            ("_TZE200_68nvbio9", "TS0601"),
            ("_TZE200_ergbiejo", "TS0601"),
            ("_TZE200_nhyj64w2", "TS0601"),
            ("_TZE200_cf1sl3tj", "TS0601"),
            ("_TZE200_7eue9vhc", "TS0601"),
            ("_TZE200_bv1jcqqu", "TS0601"),
            ("_TZE200_nw1r9hp6", "TS0601"),
            ("_TZE200_9p5xmj5r", "TS0601"),
        ],
        ENDPOINTS: {
            1: {
                PROFILE_ID: zha.PROFILE_ID,
                DEVICE_TYPE: zha.DeviceType.SMART_PLUG,
                INPUT_CLUSTERS: [
                    Basic.cluster_id,
                    Groups.cluster_id,
                    Scenes.cluster_id,
                    TuyaManufCluster.cluster_id,
                ],
                OUTPUT_CLUSTERS: [Time.cluster_id, Ota.cluster_id],
            }
        },
    }

    replacement = {
        ENDPOINTS: {
            1: {
                DEVICE_TYPE: zha.DeviceType.WINDOW_COVERING_DEVICE,
                INPUT_CLUSTERS: [
                    Basic.cluster_id,
                    Groups.cluster_id,
                    Scenes.cluster_id,
                    TuyaManufacturerWindowCover,
                    TuyaWindowCoverControl,
                ],
                OUTPUT_CLUSTERS: [Time.cluster_id, Ota.cluster_id],
            }
        }
    }`

Reply:
### Great done !!!
If its working OK can you pleas making one PR for getting it added in (Z)HA so all users can using  it  without using local quirk that can being broken in every ZHA release ?

Reply:
OK, PR is updated. i moved it to the correct inverted class and added the required comma